packages/server-nodejs/src/JitarServer.ts

```diff
@@ -36,20 +35,25 @@ const STARTUP_MESSAGE = `
   By Masking Technology (masking.tech)
 `;
 
-export default class JitarServer extends OvernightServer
+export default class JitarServer
 {
+    #app: Express;
+    //#router: express.Router;
```

Comment:
I don't think the router will come back...

Reply:
Done